samples/openapi3/client/petstore/python-experimental/tests/test_api_validation.py

```diff
@@ -42,6 +43,12 @@ def test_configuration(self):
             config.disabled_client_side_validations = 'foo'
         config.disabled_client_side_validations = ""
 
+    def test_array_of_enums(self):
+        data = [
+            "placed", None
+        ]
+        response = MockResponse(data=json.dumps(data))
+        deserialized = self.api_client.deserialize(response, (array_of_enums.ArrayOfEnums, ), True)
```

Comment:
@spacether this does not work. I am going with following hack meantime:
```suggestion
        deserialized = self.api_client.deserialize(response, (list, ), True)
```